<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>561</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/ico_UHelper.ico</normaloff>:/ico_UHelper.ico</iconset>
  </property>
  <widget class="QWidget" name="UHelperwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,4">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_uartSetting">
          <property name="title">
           <string>串口设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="0">
            <widget class="QLabel" name="label_uartStopBit">
             <property name="text">
              <string>停止位</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_uartFlowControl">
             <property name="text">
              <string>流控</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_uartCheckBit">
             <property name="text">
              <string>校验位</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="comBox_uartStopBit">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comBox_uartDataLen">
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comBox_uartDps">
             <item>
              <property name="text">
               <string>1200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4800</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>38400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>57600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>115200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Custom</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_uartPort">
             <property name="text">
              <string>端口</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_uartDataLen">
             <property name="text">
              <string>数据位</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_uartDps">
             <property name="text">
              <string>波特率</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="MyComboBox" name="comBox_uartPort"/>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="comBox_uartCheckBit">
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Even</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Odd</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Space</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mark</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="comBox_uartFlowControl">
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>RTS/CTS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>XON/XOFF</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_uartConnect">
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_uartDisconnect">
            <property name="text">
             <string>断开</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_displaySetting">
          <property name="title">
           <string>接收设置</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButton_displayASCII">
             <property name="text">
              <string>ASCII</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="radioButton_displayHEX">
             <property name="text">
              <string>Hex</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox_displayNewline">
             <property name="text">
              <string>自动换行</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_displayTime">
             <property name="text">
              <string>显示时间</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_clear_TBinput">
             <property name="text">
              <string>清除接收</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_sendoutSetting">
          <property name="title">
           <string>发送设置</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButton_sendoutASCII">
             <property name="text">
              <string>ASCII</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="radioButton_sendoutHEX">
             <property name="text">
              <string>Hex</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox_sendoutAutoResend">
             <property name="text">
              <string>自动重发(ms)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_AutoResend">
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_clear_TBoutput">
             <property name="text">
              <string>清除发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>4</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="3,1" rowminimumheight="0,0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser_intput">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="5,0">
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <widget class="QPushButton" name="buttom_sendout">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPlainTextEdit" name="textEdit_output">
            <property name="maximumSize">
             <size>
              <width>1952222</width>
              <height>1952222</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>596</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_saveRecnews"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_tools">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="action_ASCII"/>
    <addaction name="action_baseConversion"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_pushbug"/>
    <addaction name="action_aboutUHelper"/>
    <addaction name="action_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_setting">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="menu_setStyle">
     <property name="title">
      <string>界面风格</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/skin.svg</normaloff>:/image/skin.svg</iconset>
     </property>
     <addaction name="action_styleDark"/>
    </widget>
    <addaction name="menu_setStyle"/>
    <addaction name="action_setFont"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_tools"/>
   <addaction name="menu_setting"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_saveRecnews">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/export.svg</normaloff>:/image/export.svg</iconset>
   </property>
   <property name="text">
    <string>导出.txt</string>
   </property>
  </action>
  <action name="action_savePreferences">
   <property name="text">
    <string>保存偏好</string>
   </property>
  </action>
  <action name="action_setPreferences">
   <property name="text">
    <string>选择默认偏好</string>
   </property>
  </action>
  <action name="action_ASCII">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/asciitable.svg</normaloff>:/image/asciitable.svg</iconset>
   </property>
   <property name="text">
    <string>ASCII</string>
   </property>
  </action>
  <action name="action_baseConversion">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/convert.svg</normaloff>:/image/convert.svg</iconset>
   </property>
   <property name="text">
    <string>进制转换</string>
   </property>
  </action>
  <action name="action_aboutUHelper">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/about.svg</normaloff>:/image/about.svg</iconset>
   </property>
   <property name="text">
    <string>关于UHelper</string>
   </property>
  </action>
  <action name="action_aboutAuthor">
   <property name="text">
    <string>关于作者</string>
   </property>
  </action>
  <action name="action_pushbug">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/bug.svg</normaloff>:/image/bug.svg</iconset>
   </property>
   <property name="text">
    <string>bug反馈</string>
   </property>
  </action>
  <action name="action_setFont">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/typeface.svg</normaloff>:/image/typeface.svg</iconset>
   </property>
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="action_styleDefault">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>默认风格</string>
   </property>
  </action>
  <action name="action_styleDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>黑色炫酷</string>
   </property>
  </action>
  <action name="action_styleWhite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>白色靓丽</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/new.svg</normaloff>:/image/new.svg</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_new">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/new.svg</normaloff>:/image/new.svg</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/exit.svg</normaloff>:/image/exit.svg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/aboutqt.ico</normaloff>:/image/aboutqt.ico</iconset>
   </property>
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="action12">
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="action13">
   <property name="text">
    <string>13</string>
   </property>
  </action>
  <action name="action14">
   <property name="text">
    <string>14</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comBox_uartDps</tabstop>
  <tabstop>comBox_uartPort</tabstop>
  <tabstop>comBox_uartDataLen</tabstop>
  <tabstop>comBox_uartCheckBit</tabstop>
  <tabstop>comBox_uartStopBit</tabstop>
  <tabstop>comBox_uartFlowControl</tabstop>
  <tabstop>radioButton_displayASCII</tabstop>
  <tabstop>radioButton_displayHEX</tabstop>
  <tabstop>checkBox_displayNewline</tabstop>
  <tabstop>checkBox_displayTime</tabstop>
  <tabstop>radioButton_sendoutASCII</tabstop>
  <tabstop>radioButton_sendoutHEX</tabstop>
  <tabstop>checkBox_sendoutAutoResend</tabstop>
  <tabstop>lineEdit_AutoResend</tabstop>
  <tabstop>textEdit_output</tabstop>
  <tabstop>pushButton_clear_TBinput</tabstop>
  <tabstop>pushButton_clear_TBoutput</tabstop>
  <tabstop>textBrowser_intput</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
